### ISSUE TITLE
Nano hopper buffer for satellite pipe

### ISSUE THREAD

**Issue by loordgek** [RS485/LogisticsPipes#957]
see title

**Comment by Landrash** [comment]
A few small suggestions.
Change the post title to something like Feature Request: Nano hopper buffer upgrade for satellite pipe.

A small description as for the functionality your requesting even tough the title may explain the necessary bits. Both are in general for increased searchability and manageability of the Issues section of the repository.

**Comment by bziemons** [comment]
@Landrash nah, please do not use tags in the title, because we want to put labels on issues. That makes the title more readable. Nice to read from you anyhow :)

Other than that the issue is as bare as it could be (not a good thing in this case). Although I understand the simple request, I would rather deny it, as there were plans to get rid of the nano hopper anyway. There might be a replacement, but that's not figured out yet. @davboecki

**Comment by Landrash** [comment]
@theZorro266 Woops, I'll try to suggest that next time :p

**Comment by loordgek** [comment]
> as there were plans to get rid of the nano hopper anyway. There might be a replacement

why ???

**Comment by bziemons** [comment]
Well no offense, but it's an ugly 4 pixel thing that doesn't even make sense :smile:

**Comment by SonOfTheStars** [comment]
@theZorro266 Does that mean MK3 Crafting Capability will be removed? Or will just the Nano Hopper be replaced by something more...fancy

**Comment by 1n5aN1aC** [comment]
@theZorro266 Are you trying to tell me that Miniaturized electronics AREN'T made by soldering a bunch of larger electronics together?  :smile:

**Comment by bziemons** [comment]
@1n5aN1aC I know, I know. It sounds crazy, right?

**Comment by SonOfTheStars** [comment]
@theZorro266 I think this originally was talking about adding a MK2 Sattelite with the same Overflow handling mechanic as the MK3 Crafting Logistics Pipe...

Wich honestly, would be much aprechiated for some bulk crafting Tasks
